app/bundles/LeadBundle/Entity/Lead.php

```diff
@@ -228,6 +238,14 @@ public static function loadMetadata (ORM\ClassMetadata $metadata)
             ->cascadePersist()
             ->cascadeDetach()
             ->build();
+
+        $builder->createField('mobile', 'string')
```

Comment:
We need to wrap https://github.com/mautic/mautic/blob/9d09a084ab4cc2959a0daaeaf56f3942ac1bdcbc/app/bundles/LeadBundle/EventListener/DoctrineSubscriber.php#L58-L69 in an `if (!$table->hasColumn($f['alias'])) { // ... }`. This listener is a work around to prevent Doctrine from deleting dynamic, custom lead fields when generating the schema. Because phone and mobile are now listed in the metadata, it throws an exception that it already exists leading to doctrine wanting to drop all the custom columns when `doctrine:schema:update` is executed.